### DISCUSSION TITLE
Installing PHPUnit - Cannot open file "install"

### DISCUSSION THREAD

**Discussion by sebastian0119** [symfony/symfony#50843] [Symfony Book: The Fast Track]
Hi there,
I am just trying to start with the book, but am stuck at chapter 2.4 "[Getting the project source code](https://symfony.com/doc/6.2/the-fast-track/en/2-project.html#getting-the-project-source-code)".
Before I started, I checked the book requirements (`symfony book:check-requirements`) and everything is good to go.
But when running `symfony new --version=6.2-1 --book guestbook` and getting to "Installing PHPUnit (this might take some time)" the operation is cancelled, indicating the error "Cannot open file 'install'".
Re-running with `--debug` does not reveal any additional information. 

I am installing things locally on Ubuntu 22.04.2 LTS

Thanks in advance.
Regards,
Sebastian

**Comment by floriannicolas** [comment]
I had the same error, and my problem was that I was using an old version of `symfony-cli`.
I was on `Symfony CLI version v4.25.1`and the last version (up to date) is `Symfony CLI version 5.10.6`.
I just had to update it to make this error disappear.

You can check your version using `symfony -V` command and try `symfony self-update` to update it. In my case the self-update didn't work, so I just had to remove it then reinstall it with brew.
